### PR TITLE
Start mentioning trailers. See #884.

### PR DESCRIPTION
**Pull request by ioggstream**
## This PR

Mention trailers.

```
If received in trailers, Digest MUST NOT be discarded; instead it MAY be merged in the header section (See Section 7.1.2 of {{!MESSAGING=I-D.ietf-httpbis-messaging}}).
```

See #884

### CODE REVIEW

File: draft-ietf-httpbis-digest-headers.md [hunk 1]
```diff
@@ -282,6 +282,10 @@ A sender MAY send a representation-data-digest using a digest-algorithm without
 knowing whether the recipient supports the digest-algorithm, or even knowing
 that the recipient will ignore it.
 
+Digest can be used in trailers. When using incremental digest-algorithms
+this allows the sender and the receiver to dynamically compute the digest value
+while streaming the content.
```
Comment (LPardue) [inline draft-ietf-httpbis-digest-headers.md:287]:
```suggestion
Digest can be sent in a trailer section. When using incremental digest-algorithms
this allows the sender and the receiver to dynamically compute the digest value
while streaming the content.
```

File: draft-ietf-httpbis-digest-headers.md [hunk 2]
```diff
@@ -999,6 +1003,16 @@ the transport layer that protects HTTP fields.
 A `Digest` field using NOT RECOMMENDED digest-algorithms SHOULD NOT be used in
 signatures.
 
+## Usage in trailers
+
+When used in trailers, the receiver gets the digest value after the payload body
+and may thus be tempted to process the data before validating the digest value.
+Instead, data should be always processed after checking that Digest has the expected value.
```
Comment (LPardue) [inline draft-ietf-httpbis-digest-headers.md:1010]:
```suggestion
Instead, data should only be processed after validating the Digest.
```

do you want to make this should a normative requirement?

File: draft-ietf-httpbis-digest-headers.md [hunk 3]
```diff
@@ -999,6 +1036,20 @@ the transport layer that protects HTTP fields.
 A `Digest` field using NOT RECOMMENDED digest-algorithms SHOULD NOT be used in
 signatures.
 
+## Usage in trailers
+
+When used in trailers, the receiver gets the digest value after the payload body
+and may thus be tempted to process the data before validating the digest value.
+Instead, data should only be processed after validating the Digest.
+
+If received in trailers, Digest MUST NOT be discarded;
```
Comment (ioggstream) [inline draft-ietf-httpbis-digest-headers.md:1045]: @LPardue sorry Lucas, could you please check this part I stumbled upon re-reading http-messaging-latest?

File: draft-ietf-httpbis-digest-headers.md [hunk 4]
```diff
@@ -999,6 +1036,20 @@ the transport layer that protects HTTP fields.
...
+instead it MAY be merged in the header section (See Section 7.1.2 of {{!MESSAGING=I-D.ietf-httpbis
+-messaging}}).
```
Comment (ioggstream) [inline draft-ietf-httpbis-digest-headers.md:1047]: @LPardue sorry Lucas, could you please check this part I stumbled upon re-reading http-messaging-latest?

Comment (LPardue) [reply draft-ietf-httpbis-digest-headers.md:1047]: lgtm

**Pull request merged**